src/libexpr/flake/lockfile.hh

```diff
@@ -67,6 +67,9 @@ struct LockFile
     std::optional<FlakeRef> isUnlocked() const;
 
     bool operator ==(const LockFile & other) const;
+    // Needed for old gcc versions that don't syntethise it (like gcc 8.2.2
```

Comment:
```suggestion
    // Needed for old gcc versions that don't synthesize it (like gcc 8.2.2
```

Reply:
So much for British spellings :P

Reply:
Heck, I carefully and intentionally used British English :upside_down_face: